C/manual/ch_Transactions.docbook

```diff
@@ -1848,7 +1848,7 @@ Translators:
                       <term>split specific fields</term>
 
                       <listitem>
-                        <para>account and deposit/withdrawal as bare minimum.
+                        <para>account (may set base account), transfer account and deposit/withdrawal as bare minimum.
```

Comment:
The transfer account is optional. If not set, gnucash will attempt to guess it based on already existing transactions with or without bayesian heuristics. An account (or the base account) on the other hand is mandatory.